### PR TITLE
ADM-428  Import classification

### PR DESCRIPTION
**Pull request by tingwang1234**
## Summary

import classification settings

## After

image

If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

File: frontend/src/components/Metrics/MetricsStep/CycleTime/index.tsx [hunk 1]
```diff
@@ -6,35 +6,16 @@ import { ErrorDone } from '@src/components/Metrics/MetricsStep/CycleTime/style'
...
-      defaultOptionValue = Object.values(validImportValue)[0]
-    }
-
-    return { name: controlName, value: defaultOptionValue }
-  }
-  const defaultValue = Object.values(columns).map((item) => getDefaultOptionValue(item, importedCycleTimeSettings))
-  const isProjectCreated = useAppSelector(selectMetricsContent).isProjectCreated
-  const columnsOption = Object.values(columns).map((item) => ({ name: item.value.name, value: '' }))
-  const [cycleTimeOptions, setCycleTimeOptions] = useState(isProjectCreated ? columnsOption : defaultValue)
+  const cycleTimeSettings = useAppSelector(selectMetricsContent).cycleTimeSettings
```
Comment (fengxinhou) [inline frontend/src/components/Metrics/MetricsStep/CycleTime/index.tsx:17]: const {cycleTimeSettings} = useAppSelector(selectMetricsContent)?

File: frontend/src/context/Metrics/metricsSlice.tsx [hunk 2]
```diff
@@ -78,14 +90,41 @@ export const metricsSlice = createSlice({
...
-      state.isProjectCreated = isProjectCreated
-      state.importFile = basic
-      state.users = basic.crews || state.users
-      state.importedCycleTimeSettings = basic.cycleTime?.jiraColumns
-      state.doneColumn = basic.realDone || state.doneColumn
-      state.classification = basic.classification || state.classification
+      const { targetFields, users, jiraColumns, isProjectCreated } = action.payload
+      state.users = isProjectCreated
+        ? users
+        : users?.filter((item: string) => state.importedData.importedCrews.includes(item))
```
Comment (fengxinhou) [inline frontend/src/context/Metrics/metricsSlice.tsx:108]: maybe can use deconstruction method  state.importedData.importedCrews/state.importedData.importedClassification

File: frontend/src/components/Metrics/MetricsStep/CycleTime/index.tsx (hunk 1 above)
Comment (tingwang1234) [reply frontend/src/components/Metrics/MetricsStep/CycleTime/index.tsx]: I have fix it

File: frontend/src/context/Metrics/metricsSlice.tsx (hunk 2 above)
Comment (tingwang1234) [reply frontend/src/context/Metrics/metricsSlice.tsx]: I have fixed it

**Review by fengxinhou — APPROVED** [review]
LGTM

**Pull request merged**